components/tools/ToolsDashboard.tsx

```diff
@@ -225,23 +224,16 @@ export default function ToolsDashboard() {
             <span className='ml-3'>Clear Filters</span>
           </div>
         )}
-        {loading ? (
-          <div className='mx-auto my-24 flex w-fit animate-pulse gap-4 text-black'>
-            <img src={loader} alt='loading' className='mx-auto w-16' />
-            <div className='my-auto text-xl'>Loading Tools...</div>
-          </div>
-        ) : (
```

Comment:
We want loading inside tools dashboard or lazy loading because there are too much content to show in the tools dashboard page. Not applying any preloader will give blank screen to the user, on first interaction with page.

Reply:
Akasht, Could you please visit the deployed version of this PR? 
https://deploy-preview-3116--asyncapi-website.netlify.app/tools

We can keep the loading state, but based on the deployed version, I don't think it's necessary, as the content is instantly visible when the page loads. 

Please let me know if we still want to keep the loading state.

Reply:
Enable throttling inside your browser and you will then see a long wait with a white screen.

Reply:
My exams are around. I will give an update in few days.

Reply:
Hello @akshatnema, Thanks for the suggestion. I have tried enabling throttling, and yes I did see a long wait with a white screen. However, when I switched to the master branch, the long wait screen was still there. Please see the below recordings. 

**Browser:** Chrome (incognito tab)
**Throttling:** Fast 4G 

## On the master branch - White screen for ~9.83s
https://github.com/user-attachments/assets/ed9fb2c4-0206-481d-891d-02fa043542de

## On the fix-scroll-id branch (branch of current PR) - White screen for ~10.38s
https://github.com/user-attachments/assets/b5c7d959-fa51-4ea1-acdd-4fe2f1735e0f